roles/telemetry_logging/tasks/main.yml

```diff
@@ -3,3 +3,10 @@
   when: journal_list is defined
   ansible.builtin.include_tasks: "journal_tests.yml"
   loop: "{{ journal_list }}"
+
+#All containers need to be up in order for this task to pass
+- name: "[TEST] verify status of multiple containers"
+  ansible.builtin.include_role:
+    name: common
+    tasks_from: container_status.yml
+  loop: "{{ container_list }}"
```

Comment:
If you add the include_tasks: container_tests to common/main.yml, you can skip the loop here, anmd ahve something like

```
- name: "TEST verify status of multiple containers"
  ansible.builtin.include_role:
      name: common
  vars:
      common_container_list:  "{{ container_list }}"
      
```